### PR TITLE
Circularize Icenia 0,0 claims

### PR DESCRIPTION
**Pull request by AngrySoundTech**
many points very accurate yes

### CODE REVIEW

**Comment by AngrySoundTech** [comment]
Uh oh I deleted nyasaland

File: land_claims.civmap.json [hunk 1]
```diff
@@ -123,8 +123,7 @@
...
...1],[2526,-6664],[2574,-6605],[2575,-6373],[2487,-6318],[2476,-6265],[2359,-6099],[2061,-5966],[1734,-6033],[1432,-5666],[1311,-4800],[1376,-4764],[1492,-4853],[1534,-4831],[1596,-4852],[1734,-4811],[1810,-4851],[1925,-4665],[1941,-4645],[1946,-4596],[1954,-4538],[1891,-4474],[1855,-4425],[1800,-4427],[1734,-4354],[1737,-4208],[1679,-4134],[1709,-3949],[1855,-3716],[1868,-3540],[2220,-3380],[2559,-2891],[2659,-2879],[2719,-2890],[2763,-2856],[2816,-2886],[2868,-2900],[2974,-2900],[2999,-2860],[2966,-2810],[2951,-2724],[2840,-2789],[2742,-2685],[2713,-2613],[2677,-2596],[2678,-2489],[2700,-2457],[2757,-2440],[2820,-2456],[2848,-2448],[2882,-2373],[3051,-2331],[3078,-2243],[3073,-2182],[2969,-2161],[2964,-2099],[2900,-2033],[2896,-1965],[2929,-1874],[2940,-1754],[2965,-1705],[3040,-1686],[3069,-1690],[3107,-1634],[3107,-1563],[3151,-1539],[3253,-1510],[3362,-1555],[3375,-1582],[3557,-1580],[3627,-1577],[3832,-1515],[4106,-1312],[4154,-1052],[4574,-890],[4892,-962],[4954,-1204],[4920,-1468],[5026,-1868],[5618,-2212],[5566,-2764],[5540,-3316],[5834,-3518],[5858,-3572],[5862,-3684],[5582,-4034],[5532,-4284],[5640,-4528],[5960,-5152],[6072,-6076],[5148,-6228],[4570,-6349],[4531,-6327]]]},
```
Comment (RedDevel2) [inline land_claims.civmap.json:127]: Did you delete Nyasaland?

**Comment by Gjum** [comment]
I am so sorry but `{"x":0,"z":0,"radius":uhh 150 I guess}`

also why did you change nyasaland, also why did you not check with jsonlint to detect the missing trailing comma ;)

**Comment by AngrySoundTech** [comment]
@Gjum sorry I did it from vim on my phone

**Comment by AngrySoundTech** [comment]
also circle breaks my ray casting pls no i'll cry

**Comment by Gjum** [comment]
I mean we can keep it as polygon too I don't care really... but why would you do raycasting on land claim polygons?!

**Comment by AngrySoundTech** [comment]
I use raycasting in my bot to determine what polygons someone is inside of at any coordinates

**Pull request merged**